rpm2cpio/README.md

```diff
@@ -0,0 +1,17 @@
+# Habitat package: rpm2cpio
+
+## Description
+
+Why does the world need another rpm2cpio?  Because the existing one won't build unless you have half a ton of things that aren't really required for it, since it uses the same library used to extract RPM's.
+
+## Usage
+
+Dumps the contents of an RPM to stdout as a GNU cpio archive
+
+`rpm2cpio RPM_FILE | cpio`
```

Comment:
You probably want this to say `hab pkg exec core/rpm2cpio rpm2cpio RPM_FILE | cpio` to make it clear that you don't need to binlink it